### PR TITLE
Improve Completion through better option and file completion

### PR DESCRIPTION
**Pull request by qoomon**
Improve Completion through better option and file completion

### CODE REVIEW

File: contrib/zsh-completion/_packer [hunk 1]
```diff
@@ -1,70 +1,66 @@
-#compdef packer
```
Comment (mwhooker) [inline contrib/zsh-completion/_packer]: I think this is needed

Comment (qoomon) [inline contrib/zsh-completion/_packer]: Works for me without this line, however it works even with this line.

Comment (mwhooker) [inline contrib/zsh-completion/_packer]: hmm, I wasn't able to get it to work with

```
rm ~/.zcompdump*
fpath=(./contrib/zsh-completion/ $fpath)
autoload -U compinit
compinit
zstyle ':completion:*' menu select=2
```

but the previous version did work

**Comment by mwhooker** [comment]
works for me now, thanks!

**Pull request merged**